src/axolotl/utils/trainer.py

```diff
@@ -171,25 +172,42 @@ def add_length(sample):
     return sample
 
 
-def drop_long_seq(sample, sequence_len=2048, min_sequence_len=2):
+def drop_long_seq(sample, sequence_len=2048, min_sequence_len=None):
+    min_sequence_len = min_sequence_len or 2
```

Comment:
If we're defaulting here, should we just set the default within https://github.com/axolotl-ai-cloud/axolotl/blob/3915abee4cba364a83f18cc4b8c2bb571cce3bac/src/axolotl/utils/config/models/input/v0_4_1/__init__.py#L688 to be more explicit?